js/intro.js

```diff
@@ -0,0 +1,320 @@
+import { BASE_URL } from './constants.js';
+
+const notAuthorized = document.querySelector('.not-authorized-page');
+const notFound = document.querySelector('.not-found-page');
+const mainContainer = document.querySelector('.intro-main');
+const loading = document.getElementById('loading');
+
+async function makeApiCall(
+  url,
+  method = 'get',
+  body = null,
+  credentials = 'include',
+  headers = { 'content-type': 'application/json' },
+  options = null,
+) {
+  try {
+    const response = await fetch(url, {
+      method,
+      body,
+      headers,
+      credentials,
+      ...options,
+    });
+    return response;
+  } catch (err) {
+    throw err;
+  }
+}
+
+function createElement({ type, classList = [], id }) {
+  const element = document.createElement(type);
+  element.classList.add(...classList);
+  element.id = id;
+  return element;
+}
+
+function getUserId() {
+  const currentUrl = window.location.href;
```

Comment:
```js
const urlParams = new URLSearchParams(window.location.search);
urlParams.get('userId');
```

You have to do something similar to this, please check **URLSearchParams** over the mdn.

Reply:
This is a possible way, but might not be a good way.

Reply:
I will do this after talk to ankush that he need in queryparameter userId or not...right now he is guiding me how to make route in morning session